docs/pages/includes/dynamodb-iam-policy.mdx

```diff
@@ -1,3 +1,99 @@
+On startup, the Teleport Auth Service checks whether the DynamoDB table you have
+specified in its configuration file exists. If the table does not exist, the
+Auth Service attempts to create one.
+
+The IAM permissions that the Auth Service requires to manage DynamoDB tables
+depends on whether you expect to create a table yourself or enable the Auth
+Service to create and configure one for you:
+
+<Tabs>
+<TabItem label="Manage a Table Yourself">
+
+If you choose to manage a DynamoDB table yourself, the table must have the
+following attribute definitions:
+
+|Name|Type|
+|---|---|
+|`HashKey`|`S`|
+|`FullPath`|`S`|
+
+The table must also have the following key schema elements:
+
+|Name|Type|
+|---|---|
+|`HashKey`|`HASH`|
+|`FullPath`|`RANGE`|
```

Comment:
I'm quite wary of putting this specific information in the guide as they it may not be updated if we do a schema migration in future. We got burnt badly by this in the past with Terraform (https://github.com/gravitational/teleport/issues/7306)

I wonder whether maybe we could link to the code which generates the tables instead so people can read for themselves if they insist on having full control?

Reply:
I can see why we would need to prevent the impression that these schema requirements are permanent, but for me, docs should prevent users from needing to look up the source. I'm not sure, for example, if the source for generating the tables will always be in a single place or easy to follow. 

What would you think of advising users to set up the backend (and upgrade their clusters) in a non-prod environment first to ensure that schema changes don't impact users?

Reply:
@webvictim Just following up on this thread. Thanks!

Reply:
👍 This change is meant to accommodate admins who are worried about infrastructure drift (https://github.com/gravitational/teleport/issues/19974) by having Teleport create tables. I'll add a comment in the source